ios/RCTWebRTC/RTCVideoViewManager.m

```diff
@@ -314,6 +316,38 @@ - (void)setVideoTrack:(RTCVideoTrack *)videoTrack {
 
     _videoTrack = videoTrack;
 
+    // Clear the videoView by rendering a 1x1 blank frame.
+    CVPixelBufferRef pixelBuffer;
+    CVReturn err = CVPixelBufferCreate(NULL, 1, 1, kCVPixelFormatType_32BGRA, NULL, &pixelBuffer);
+    if (err != kCVReturnSuccess) {
```

Comment:
Is this error handling correct? We are only doing stuff if we don't succeed?

Reply:
Whoops! Thanks for catching this.